spikeinterface/sorters/runsorter.py

```diff
@@ -50,15 +50,15 @@
     raise_error: bool
         If True, an error is raised if spike sorting fails (default).
         If False, the process continues and the error is logged in the log file.
-    docker_image: bool, str, or None
+    docker_image: bool or str
```

Comment:
Why did we loose the logic of None. I prefer the logic None vs something rather than False vs something.